controllers/ControllerAuth.php

```diff
@@ -44,7 +37,7 @@ public function login() {
 
 	public function logout(){
 		session_destroy();
-		header("Location: http://".$this->_server."/Home");
+		header("Location: http://".$this->_server);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)